spec/02-integration/08-status_api/01-core_routes_spec.lua

```diff
@@ -40,7 +40,48 @@ describe("Status API - with strategy #" .. strategy, function()
       assert.is_number(json.server.connections_writing)
       assert.is_number(json.server.connections_waiting)
       assert.is_number(json.server.total_requests)
+      assert.is_nil(json.server.configuration_hash) -- no hash in DB more, or in DBLESS mode until configuration has been applied
     end)
+
+    it("/status starts providing a configuration_hash once an initial configuration has been pushed in dbless mode #off", function()
+      -- push an initial configuration so that a configuration_hash will be present
+      local postres = assert(client:send {
+        method = "POST",
+        path = "/config",
+        body = {
+          config = [[
+          _format_version: "1.1"
+          services:
+          - host = "konghq.com"
+          ]],
+        },
+        headers = {
+          ["Content-Type"] = "application/json"
+        }
+      })
+      assert.res_status(201, postres)
+
+      -- verify the status endpoint now includes a value (other than the default) for the configuration_hash
```

Comment:
This codebase generally avoids such comments since they get out of date.
The best way to capture the intent is in the description of the tests.

Reply:
```suggestion
```